charts/kong/templates/_helpers.tpl

```diff
@@ -56,6 +56,122 @@ Create the name of the service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create Ingress resource for a Kong service
+*/}}
+{{- define "kong.ingress" -}}
+{{- $servicePort := include "kong.ingress.servicePort" . }}
+{{- $path := .ingress.path -}}
+{{- $hostname := .ingress.hostname -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .fullName }}-{{ .serviceName }}
+  namespace: {{ .namespace }}
+  labels:
+   {{- .metaLabels | nindent 4 }}
+ {{- if .ingress.annotations }}
+  annotations:
+ {{- range $key, $value := .ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+ {{- end }}
+ {{- end }}
```

Comment:
Indentation of `{{}}` blocks under this resource appears off by 1 space (at least here and on line 88).